extensions/sql-migration/src/models/stateMachine.ts

```diff
@@ -688,7 +688,7 @@ export class MigrationStateModel implements Model, vscode.Disposable {
 		}
 	}
 
-	public async startSkuTimers(page: SKURecommendationPage, refreshIntervalInMs: number): Promise<void> {
+	public async startSkuTimers(page: SKURecommendationPage): Promise<void> {
```

Comment:
these timers should always be on fixed intervals (query every 30 seconds, run recommendation every 10 minutes) and having a variable `refreshIntervalInMs` may have been the original issue, so this section was simplified